README.md

```diff
@@ -118,7 +118,14 @@ and then register a URL type entry using the value of `CFBundleIdentifier` as th
     </dict>
 </array>
 ```
+the end result should include _just_ the Bundle Identifier (ex: `samples.auth0.com`), and so look something like this:
```

Comment:
It might be clearer to incorporate something along the lines of 
-> the string value should be the literal value of the bundle identifier with no **$** variables

Reply:
that works!! as someone who (prior to RN) never touched native code and is coming from web dev i think it's helpful for it to be super explicit, that was my intent here. thx.!

Reply:
Agreed, if you can fix and commit and I'll merge it. Thanks

Reply:
hey there it's not letting me edit this again, since i didn't make a branch or whatever, thanks.

Reply:
I don't think I can do anything with it either now, so either fork it and make a new PR or I'll do it myself in a bit.

Reply:
lol so silly, here's this: https://github.com/auth0/react-native-auth0/pull/106 thanks again!